test/unittests/iotools/converters/adni_to_bids/test_adni_utils.py

```diff
@@ -5,6 +5,55 @@
 from pandas.testing import assert_frame_equal, assert_series_equal
 
 
+@pytest.mark.parametrize(
+    "input, expected",
+    [
+        (
+            {"001_S_0001", "001_S_0002", "001_S_0003"},
+            {"001_S_0001", "001_S_0002", "001_S_0003"},
+        ),
+        ({"001_S_0001", "001_S_00014", ".001_S_0001", "001S0001"}, {"001_S_0001"}),
```

Comment:
I might be missing something but I don't understand why you need to parametrize.
Moreover, looking at this, it feels like the same input produces different expectations (which is not what you're testing, but still a bit misleading...).

Maybe split into two test functions without any parametrization. One will test with a provided subject list, and the other one without.
WDYT ?

Reply:
Thanks for the review ! You're right that I should have split in two test functions (list or without), so I changed that. However I still wanted to check that the regex I use to get the subjects from the directory when there is no list is working well, which is why I put a parametrization. Now that the functions are split it might be a bit clearer. 

WDYT ?

Reply:
Hi @AliceJoubert !

Re-reading this with a rested mind, I think this is fine like this. Sorry for the noise.

Reply:
Alright, no worries :-) ! For clarity I'm thinking of leaving the tests as is (split test). Do you see anything else ?

Reply:
Sounds good ! Nope, I think this is ready to merge ! 👍